build.gradle

```diff
@@ -10,22 +10,18 @@
  */
 
 /*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
  *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
```

Comment:
Is the change to license header intended?
We do not want to change any existing license headers.

Reply:
I made the changes in license header following this example: https://github.com/opensearch-project/anomaly-detection/blob/main/build.gradle#L13-L25. If there's no need to change it, I'll undo it.

Reply:
Good question, Sure the existing headers are not to be modified. This is the guidance from Legal.
So leave it as it was before. You already have the SPDX header on top which is perfect.

Reply:
I see. I'll undo the changes.

Reply:
I've reverted the changes in license header.